index.js

```diff
@@ -46,8 +46,8 @@ cs.get.rgb = function (string) {
 		return null;
 	}
 
-	var abbr = /^#([a-fA-F0-9]{3})$/;
-	var hex = /^#([a-fA-F0-9]{6})$/;
+	var abbr = /^#([a-fA-F0-9]{3,4})$/;
```

Comment:
do you want me to match the model of the regex you wrote in the comment below? something like:

`/^#([a-f0-9]{3})([a-f0-9])?$/i`?